fix: card style

dipanshurdev opened the pull request:
## Description
Please update the CSS for the Style Carousel Card to adjust the width and height.

Fixes # (issue)

#752 

## Type of Change

- [ ] New feature (e.g., new page, component, or functionality)
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] UI/UX improvement (design, layout, or styling updates)
- [ ] Performance optimization (e.g., code splitting, caching)
- [ ] Documentation update (README, contribution guidelines, etc.)
- [ ] Other (please specify):

## Preview
<img width="1903" height="589" alt="Screenshot 2025-10-04 142713" src="https://github.com/user-attachments/assets/f2539be0-05ba-4d62-9547-d9c6f388df38" />

sanjay-kv commented:
<img width="1825" height="610" alt="image" src="https://github.com/user-attachments/assets/97afcf2f-f26e-4931-a4bc-704480a79ab2" />

The gap between content is two much , the red box is ideal height expecting.

dipanshurdev commented:
> Hi @dipanshurdev , could you share a short video ? it would be better to review .

Hi @Adez017 , Here is an overview: 

On TestimonialCard, i have updated the CSS, removed extra padding & margin, and added max-height: 
<img width="1168" height="365" alt="Screenshot 2025-10-06 202325" src="https://github.com/user-attachments/assets/48141b1b-59f5-43ce-bb4b-6796eb636c48" />

Adez017 reviewed (changes requested):
try to fix the logo's on the testimonials , in your changes it seems to be very small and doesnt  feel UI friendly .

Adez017 reviewed (approved):
looks good 👍
